sbc_compassion/model/partner_compassion.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emmanuel Mathier <emmanuel.mathier@gmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+
+    """ This class upgrade the partners to match Compassion needs.
+    """
+
+    _inherit = 'res.partner'
+
+    ##########################################################################
+    #                        NEW PARTNER FIELDS                              #
+    ##########################################################################
+
+    spoken_langs = fields.Many2many('res.lang.compassion')
```

Comment:
The naming convention for relation fields is to end with `_id` or `_ids` suffixes. Here it would be `spoken_lang_ids`